src/test/scala/io/pravega/connectors/spark/PravegaMicroBatchSourceSuite.scala

```diff
@@ -250,6 +286,7 @@ abstract class PravegaSourceSuiteBase extends PravegaSourceTest {
       .option(SCOPE_OPTION_KEY, testUtils.scope)
       .option(STREAM_OPTION_KEY, streamName)
       .option(START_STREAM_CUT_OPTION_KEY, STREAM_CUT_EARLIEST)
+      .option(MAX_OFFSET_PER_TRIGGER, "1")
```

Comment:
We do not need this change for normal test cases, right?

Reply:
done.
Added separate tests for maxOffset, original test cases are kept as it is.